website/other/bezier-rs-demos/src/main.ts

```diff
@@ -32,31 +17,92 @@ declare global {
 	}
 }
 
+window.document.title = "Bezier-rs Interactive Documentation";
+
 window.customElements.define("bezier-demo", BezierDemo);
 window.customElements.define("bezier-demo-pane", BezierDemoPane);
 window.customElements.define("subpath-demo", SubpathDemo);
 window.customElements.define("subpath-demo-pane", SubpathDemoPane);
 
-const bezierDemos = document.getElementById("bezier-demos");
-(Object.keys(bezierFeatures) as BezierFeatureName[]).forEach((featureName) => {
+function renderBezierPane(featureName: BezierFeatureKey, container: HTMLElement | null): void {
 	const feature = bezierFeatures[featureName];
 	const demo = document.createElement("bezier-demo-pane");
 
 	demo.setAttribute("name", featureName);
 	demo.setAttribute("demoOptions", JSON.stringify(feature.demoOptions || {}));
 	demo.setAttribute("triggerOnMouseMove", String(feature.triggerOnMouseMove));
 	demo.setAttribute("chooseComputeType", String(feature.chooseComputeType));
-	bezierDemos?.append(demo);
-});
+	container?.append(demo);
+}
 
-const subpathDemos = document.getElementById("subpath-demos");
-(Object.keys(subpathFeatures) as SubpathFeatureName[]).forEach((featureName) => {
+function renderSubpathPane(featureName: SubpathFeatureKey, container: HTMLElement | null): void {
 	const feature = subpathFeatures[featureName];
 	const demo = document.createElement("subpath-demo-pane");
 
 	demo.setAttribute("name", featureName);
 	demo.setAttribute("sliderOptions", JSON.stringify(feature.sliderOptions || []));
 	demo.setAttribute("triggerOnMouseMove", String(feature.triggerOnMouseMove));
 	demo.setAttribute("chooseComputeType", String(feature.chooseComputeType));
-	subpathDemos?.append(demo);
+	container?.append(demo);
+}
+
+function isUrlSolo(url: string): boolean {
+	const hash = url.split("#")?.[1];
+	const splitHash = hash?.split("_");
```

Comment:
Please use a slash instead of an underscore. Also, instead of the camelCase IDs, it would be nice to use kebab-case. That probably means changing the keys in you bezier-features.ts and subpath-features.ts.

Reply:
The current url format is: `<beizer | subpath>_<idInCamelCase>` with an optional `<_solo>` at the end. If we change everything to be separated by a hyphen, it'll be harder to see the id in the url. For example: `bezier-bezier-through-points` looks a little weird since the word bezier is repeated. I just want to confirm that this is ok.

Reply:
Oh oops, sorry i misread that!